### DISCUSSION TITLE
[Question or Bug] - a docs() commit with breaking changes introduced major bump and release

### DISCUSSION THREAD

**Discussion by sag1v** [semantic-release/semantic-release#1927] [Commit Convention] [answered]
## Current behavior
I'm not sure if this is by design but even so i could use an advice on how to solve it:
Someone on my team created a `docs()` commit (which shouldn't change the version or trigger a release) but accidentally wrote "None" as Breaking Changes. So i guess semantic-release analyzer thinks there are breaking changes, and bumped a major version and triggered a release (private artifactory). 

There is no problem for us to remove the release from artifactory (already done) but we want to revert the version change and go back to the previous version, and obviously keep the changes made to the docs.

What is sane way to achieve this? 

## Expected behavior
Maybe its better that semantic-release will ignore the Breaking Changes when the commit message is docs? i don't know 🤷‍♂️ 

## Environment

- **semantic-release** version: 
    ```"@semantic-release/changelog": "^5.0.1",
    "@semantic-release/exec": "^5.0.0",
    "@semantic-release/git": "^9.0.0",
    "@semantic-release/gitlab": "^6.0.4",
    "@semantic-release/npm": "^7.0.6",```

- CI environment: GitLab CI
- Plugins used: <!-- List semantic-release plugin used if any -->
- **semantic-release** configuration: private repo
- CI logs: <!-- link to your CI logs or semantic-release logs -->

**Comment by travi** [comment, marked as answer]
> I'm not sure if this is by design

the commit analyzer follows the convention very strictly. in the default convention, it is valid for any commit type to include breaking changes, which is the reason these are specified in the footer of the commit body rather than as part of the commit subject. please consult the documentation of the convention used by your project to understand the details of what to expect 

> accidentally wrote "None" as Breaking Changes

it would potentially be helpful to understand how this was done accidentally. in the default convention, the footer of the commit must contain the very specific string of `BREAKING CHANGE: ` in order for the parser to find such content. if using a tool like commitizen, the default adapter first asks if there are breaking changes before enabling the developer to input such content

> There is no problem for us to remove the release from artifactory (already done) but we want to revert the version change and go back to the previous version, and obviously keep the changes made to the docs.

we do not ever recommend unpublishing versions. i understand that this level of bump was not intentional, but versions are simply to inform consumers of the impact of the changes. it is better to suggest that the impact is higher than it is than the other way around. we recommend simply moving on in cases like you found yourself. 

however, since it appears that you have already unpublished the version from your registry, you likely want to do one of two things:
* remove the tag from version control so that it matches what is in your registry. this would result in semantic-release re-evaluating these commits next time, resulting in the same type of publish unless you rewrite history to remove the `BREAKING CHANGE: ` from that commit
* leave the tag in version control so that semantic-release does not re-evaluate these commits and simply publishes the next version as normal (but still being after the major version bump)

> Maybe its better that semantic-release will ignore the Breaking Changes when the commit message is docs?

this is not the behavior expected by the commit convention, so semantic-release will not stray from what is defined there. if you want different behavior, you need to define a different convention for semantic-release to follow

**Reply by sag1v** [reply]
Thanks for the answer, was making sure we are not missing anything
